base/applications/sndvol32/lang/it-IT.rc

```diff
@@ -113,24 +113,24 @@ END
 
 IDD_ADVANCED DIALOGEX 0, 0, 255, 140
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Advanced Controls for Volume Control"
+CAPTION "Controlli avanzati per il volume"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL IDI_MAINAPP, -1, "static", SS_ICON | WS_CHILD | WS_VISIBLE, 7, 7, 32, 32
-    LTEXT "These settings can be used to make fine adjustments to your audio.", -1, 44, 7, 200, 32
-    GROUPBOX "Tone Controls", IDC_ADV_TONE_CONTROLS, 7, 33, 240, 80
-    LTEXT "These settings control how the tone of your audio sounds.", -1, 17, 48, 200, 14
+    LTEXT "Queste impostazioni possono essere usate per fare regolazioni fini al tuo audio.", -1, 44, 7, 200, 32
+    GROUPBOX "Controlli di tono", IDC_ADV_TONE_CONTROLS, 7, 33, 240, 80
+    LTEXT "Queste impostazioni controllano il modo in cui suona il tono del tuo audio.", -1, 17, 48, 200, 14
     LTEXT "&Bass:", -1, 17, 62, 50, 8
```

Comment:
Maybe these words would be more appropriate:

**Basso**: _Quiete_ / _Louder_
**Alto**: _Quiete_ / _Louder_

Currently you have:

Bass: **Basso** / **Alto**
Treble: **Basso** / **Alto**

Cc @Fraizeraust and @ElCresh

Reply:
For **Bass** and **Treble** the author could just use `Basso` for Bass and `Alto` for Treble (with further indicating that these are range tones and not related with High/Low).

Reply:
In italian we indicate freq with "Alti" (plural!) (Trebble) and "Bassi" (plural) (Bass) and the volume with "Basso" (singular) and "Alto" (singular). So a mix of the two is the right answers.